datasources/nm.ejs

```diff
@@ -17,7 +17,7 @@
   ],
   "catalog": [
         {
-            "name": "National Data Sets",
+            "name": "National Datasets",
```

Comment:
ok, thanks, I'll revert that and then merge the rest.  And write an issue for the problem keeping us from being able to do a rename like this.